tox.ini

```
@@ -57,10 +57,12 @@ commands =
 allowlist_externals = ./scripts/ruff.sh
 
 [testenv:spellcheck]
-description = spell check (needs 'aspell' or 'hunspell' command)
+description = spell check (needs 'aspell' command)
 skip_install = true
 skipsdist = true
 deps =
     pyspelling
 commands =
-    {envpython} -m pyspelling --config {toxinidir}/.spellcheck.yml
+    bash -c 'command -v aspell || (echo "aspell is not installed. Please install it." && exit 1)'
```

```suggestion
    sh -c 'command -v aspell || (echo "aspell is not installed. Please install it." && exit 1)'
```
+1 for using `command` instead of `which`